src/main/flight/mixer.c

```diff
@@ -529,6 +530,12 @@ void FAST_CODE mixTable(const float dT)
 
     throttleRange = throttleMax - throttleMin;
 
+    // Throttle Percent used by OSD and MSP
+    if(navigationIsControllingThrottle())
```

Comment:
I was rather hoping we could get a function in Mixer to compute this and use when needed in other places

Reply:
Ok, fixed as requested.